odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -2018,3 +2020,29 @@ class ModelAutovacuumed(models.Model):
     @api.autovacuum
     def _gc(self):
         self.search([('expire_at', '<', datetime.datetime.now() - datetime.timedelta(days=1))]).unlink()
+
+
+class ShareComputeMethod(models.Model):
+    _name = _description = 'test_new_api.shared.compute'
+
+    name = fields.Char(compute='_compute_name', store=True, readonly=False)
+    start_datetime = fields.Datetime(compute='_compute_datetime', store=True, readonly=False)
+    end_datetime = fields.Datetime(compute='_compute_datetime', store=True, readonly=False)
```

Comment:
Using datetime fields seems to bring unnecessary complexity (freezegun and datetime conversions).
Why don't you simply use three character fields where the computations ensure something like `field1 = field2 + "." + field3`?
That being said, kudos for the test :+1: